android/app/src/main/AndroidManifest.xml

```diff
@@ -51,9 +51,6 @@
                  screen fades out. A splash screen is useful to avoid any visual
                  gap between the end of Android's launch screen and the painting of
                  Flutter's first frame. -->
```

Comment:
Why have you deleted this file?.
Also this comment it part of the line that you deleted, it should get removed with it